config/settings.yml

```
@@ -4,3 +4,5 @@ length:
   digit_10: 10
   digit_20: 20
   digit_255: 255
+expired:
+  password_reset: 2
```

```suggestion
  reset_2_hours: 2
```
nên đặt tên biến thế này để mình dễ dàng tái sử dụng khi cần giá trị 2  và cả đọc tên biến là hiểu ý nghĩa cu nhé

---

Dạ vâng chị, e hiểu rồi